### ISSUE TITLE
C_A matrix

### ISSUE THREAD

**Issue by dc5048** [cybergalactic/MSS#17]
https://github.com/cybergalactic/MSS/blob/04282b4ebe7ac3dfcda9effbb139642693606ccd/VESSELS/remus100.m#L106

Hello Dr. Fossen,
First off, thanks very much for your excellent textbook and this accompanying toolkit.  I've been using them as my primary reference for a model of dynamics and control of a small AUV, similar in concept to the Remus 100.  I'm wondering, should this call to imlay61 pass in the relative velocity vector nu_r instead of nu? I'm looking at equation 6.59 and 6.60 from your text, which express MA and CA purely in terms of nu_r, with MA assumed to be a diagonal matrix as it is in imlay61.m

The reason I ask is that when CA is calculated this way using the absolute velocity, the terms in the lower left quadrant of CA cause angular moments that are dependent on water current.  For example, if the vessel were drifting with a fast water current (zero relative velocity) that had non-zero u,v,and w components in the body frame, substantial angular moments arise from CA which doesn't seem intuitively correct. 

Thanks again,
DC

**Comment by cybergalactic** [comment]
Yes, this is a bug. Imlay61.m should be called with the relative velocity. I have updated the m-file on GitHub. Thank you for pointing this out.  Also, be aware that the remus100.m function assumes that d/dt nu_r = 0. A further improvement could be to use a non-constant relative velocity. I will update this later. Kind Regards Thor I. Fossen
